app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -4,10 +4,6 @@ heading: "Train to teach if you do not have a degree"
 subcategory: Other routes into teaching
 description: |-
   Explore how you can get into teaching without a degree by taking an undergraduate course to gain qualified teacher status (QTS) alongside a bachelor's degree.
-related_content:
-    Explore life as a teacher : "/life-as-a-teacher"
-    Funding your training : "/funding-and-support"
-    Teach in further education without a degree : "https://www.teachinfurthereducation.education.gov.uk/"
```

Comment:
I wonder if we should add a very small section on FE and EY right at the end to support cross-workforce join-up?

Could use the wording from the routes wizard?